packages/checkout/widgets-lib/src/components/Transactions/ChangeWallet.tsx

```diff
@@ -39,18 +49,38 @@ export function ChangeWallet({ onChangeWalletClick }: ChangeWalletProps) {
     onChangeWalletClick();
   };
 
+  useEffect(() => {
+    if (isWalletConnectEnabled) {
+      setIsWalletConnect((from?.web3Provider.provider as any)?.isWalletConnect);
```

Comment:
Just FYI we do have util functions for checking these, isMetaMaskProvider(provider), isPassportProvider(provider) and isWalletConnectProvider(provider)

Reply:
Oh nice, I didn't realise! Might make sense to add an isWalletConnectProvider(provider) util as well?